crates/net/network/src/transactions.rs

```diff
@@ -158,6 +161,15 @@ where
         TransactionsHandle { manager_tx: self.command_tx.clone() }
     }
 
+    /// Returns true if the node has synced with the network at least once.
+    fn has_synced(&mut self) -> bool {
+        if self.has_synced {
+            return true
+        }
+        self.has_synced = self.network.is_syncing();
```

Comment:
This is a bit weird, we set "has synced" to the vaue of "is syncing", which mean the opposite thing, should this be `!is_syncing`?